app/services/team_move_service.rb

```diff
@@ -81,10 +82,11 @@ def link_old_team_to_new_team
     @team.moved_to_unit = @new_team
     @team.save
   end
-  
+
   def restore_new_team_name_to_original_name
     # New team gets original team name to retain consistency for the users
     @new_team.name = @team.original_team_name
+    @new_team.code = @new_team.code.sub(/-NEW$/, "") unless @team.code.blank?
```

Comment:
Does this need to be gsub or case insensitive?

Reply:
Great question! I suck at regex, but I don't think so... the idea is to only replace the pattern if found at the end of the string so I think it's ok as sub rather than gsub. Similarly, we're adding the -NEW in uppercase so I only want to replace it if it's uppercase